### ISSUE TITLE
CoreMediaIO.CMIOObjectSetPropertyData ValueError

### ISSUE THREAD

**Issue by eliteraspberries** [ronaldoussoren/pyobjc#554]
**Describe the bug**

Attempting to call [CMIOObjectSetPropertyData()](https://developer.apple.com/documentation/coremediaio/1415838-cmioobjectsetpropertydata?language=objc) raises a ValueError exception:

```
  File "main.py", line 1028, in cmio_set_property
    status = CoreMediaIO.CMIOObjectSetPropertyData(
ValueError: depythonifying 'pointer', got 'CoreMediaIO.CMIOObjectPropertyAddress'
```

[CMIOObjectGetPropertyData()](https://developer.apple.com/documentation/coremediaio/1415883-cmioobjectgetpropertydata?language=objc) works fine on the same objectID.

In CoreMediaIO/_metadata.py, I see:

```
    "CMIOObjectSetPropertyData": (b"iI^{CMIOObjectPropertyAddress=III}I^vI^v",),
```

and in the same file:

```
    "CMIOObjectGetPropertyData": (
        b"iI^{CMIOObjectPropertyAddress=III}I^vI^I^v",
        "",
        {
            "arguments": {
                1: {"type_modifier": "n"},
                3: {"c_array_length_in_arg": 2, "type_modifier": "o"},
                5: {"type_modifier": "o"},
                6: {"c_array_length_in_arg": (4, 5), "type_modifier": "o"},
            }
        },
    ),
```

The two functions have the same signature except for the second to last argument.

I just copy-pasted the tuple for CMIOObjectGetPropertyData into CMIOObjectSetPropertyData, and the exception is gone, it seems to work now.

**Platform information**

* Python version: **3.8.13**
* How was python installed (python.org, anaconda, homebrew, ...): **source**
* macOS version: **13.3.1 (a)**

**To Reproduce**

See here: https://gist.github.com/eliteraspberries/0c26993bc82f7b62056cff1733ebfba8

**Expected behavior**

CMIOObjectSetPropertyData() should return non-zero (fail).

**Additional context**

**Comment by Alexandro1112** [comment]
Same error(

**Comment by ronaldoussoren** [comment]
This is due to incomplete metadata.  I'm working on a fix (starting with updating the test that should have detected this problem...)

**Comment by ronaldoussoren** [comment]
I've pushed a fix for this issue. 

I'm not sure yet when I'll push out a release with this fix, my plan was to wait for the 3.12 beta release, but that's been postponed so I may release sooner than that.
